Makefile.third

```diff
@@ -38,10 +38,13 @@ $(TURBOJPEG_LIB) $(JPEG_LIB): $(THIRDPARTY_DIR)/libjpeg-turbo/CMakeLists.txt
 	cd $(JPEG_BUILD_DIR) && \
 		$(CMAKE) \
 		-DCMAKE_VERBOSE_MAKEFILE:BOOL=OFF \
-		$(if $(findstring armv6, $(ARM_ARCH)),-DWITHOUT_SIMD:BOOL=ON,) \
+		$(if $(findstring arm, $(ARM_ARCH)),-DWITHOUT_SIMD:BOOL=ON,) \
 		-DCHOST="$(CHOST)" \
 		-DCFLAGS="$(CFLAGS)" \
 		-DLDFLAGS="$(LDFLAGS)" \
+		-DCMAKE_CXX_COMPILER="$(CMAKE_CXX_COMPILER)" \
```

Comment:
Nope, libjpeg-turbo is C only, we shouldn't need that.

If CMake complains, you have a CMake toolchain setup snafu somewhere earlier ;).